bin/controller/unittest/CCommandProcessorTest.cc

```diff
@@ -5,19 +5,19 @@
  */
 
 #include <core/CProcess.h>
-#include <core/CSleep.h>
 #include <core/CStringUtils.h>
 
 #include "../CCommandProcessor.h"
 
 #include <boost/range.hpp>
 #include <boost/test/unit_test.hpp>
 
+#include <chrono>
+#include <cstdio>
```

Comment:
There are `::remove()` calls that should be changed to `std::remove()` if we're using the C++ header.